### DISCUSSION TITLE
Onde conseguir Freelas?

### DISCUSSION THREAD

**Discussion by thiamsantos** [frontendbr/forum#198] [Discussão]
Eu estou começando na área e gostaria de dicas para conseguir Freelas.
Quais sites eu devo me cadastrar ou com mensagens quem falar? 
Preciso ter um portfólio? 
É como eu tenho 16 anos, perante a lei é permitido eu fazer Freelance? 

Obrigado desde já!!!

**Comment by willycamargo** [comment]
Não é ilegal, mas pode ter dificuldades se não emitir NF. Para emitir NF você precisa ao menos ser MEI (Micro Empreendedor Individual), para ser MEI você precisa ser de maior ou emancipado. Com 16 anos eu emancipei e consegui abrir MEI e emitir NF sem problemas.

Entretanto, gostaria de te dar uma dica baseado em minha experiência: freela não vale a pena, pelo menos para quem está no começo da carreira (que parece o seu caso). Se puder encontrar um estágio ou um emprego fixo com um mentor com mais experiência que você seria ótimo! Você poderá aprender muito e conseguirá sentir uma curva de aprendizado muito boa.

Fazer freela sem experiência pode ser ruim pra você, pode ser ruim para seu cliente, e pode ser ruim para outros desenvolvedores.

**Comment by AnthonyNalim** [comment]
Faço das palavras do amigo acima as minhas, mas caso não queira se emancipar, pode abrir no nome de algum parente que não trabalha assinar sua carteira como aprendiz e pronto. Mas teria que ler muito, acredito que possa pegar jobs pequenos em sua cidade e coisas do tipo.

Dificilmente uma empresa vai contratar um menor aprendiz pra essa área, é o que acredito, ainda mais dependendo de onde mora (interior no caso). Acredito que com 16 anos ainda esteja com seus pais, relativamente sem custo de vida, aproveitaria esse tempo ao máximo para me especializar, não ter problemas pessoas tais como (divida, sustento próprio, família pra sustentar) isso freia seu desenvolvimento, aproveita que ta sem essa carga, com muito tempo disponível e mete a cara nos estudos.

**Comment by thiamsantos** [comment]
@willycamargo e @AnthonyNalim muito obrigado pelas dicas!!!! 

Eu já pensei em pegar um estágio em uma empresa mas a questão é o tempo perdido em trânsito e como eu estou no ensino médio ainda, não sobraria tempo para estudar. Por isso a ideia de pegar Freelas, posso conciliar o tempo estudando,  ganhando experiência e ganhando um trocados de extra.

**Comment by vitocaio** [comment]
concordo com @willycamargo Muitos clientes são exigentes com a qualidade de produtos e principalmente com os prazos, se você for um desenvolvedor inexperiente, isso pode comprometer você como profissional e os outros profissionais que perderam uma oportunidade de freela, por conta disto. Acredito que seria melhor um estagio, não ocupa tanto tempo, mesmo porque estagio dura no maximo 6 horas.

**Comment by douglascorrea** [comment]
@thiamsantos "Ter um portfólio" sempre ajuda, mas acredito que o portfólio de qualquer desenvolvedor hoje em dia é o Github.

Por isso participe de projetos open source, crie seu próprio, mostre seu trabalho com "sites demo" ou no codepen (que funciona muito pra frontend).

Quanto a onde buscar trabalhos, existem vários lugares pra pegar freelas. Eu particularmente uso o [Upwork](http://www.upwork.com). Como funciona por "bids" e recomendações, no ínicio eu adotei a estratégia de fazer preços mais baratos para ter um histórico no site e hoje em dia já consigo contratos de valor hora acima de $35 usd, o que é relativamente alto se vc morar aqui no Brasil. 

Ah, mas no caso do Upwork o ingles é necessário, sempre rola entrevista, algumas vezes por chat e outras por Skype call (inclusive com camera).

**Comment by lucasvalhos** [comment]
@douglascorrea boa dica sobre o Upwork, eu utilizava muito quando era odesk.

@thiamsantos tem esse repo no github que contém uma lista de empresas que disponibilizam trabalho remoto. https://github.com/lerrua/remote-jobs-brazil

**Comment by thiamsantos** [comment]
@lucasvalhos @douglascorrea @vitocaio obrigado pela ajuda!!! 

Faz nove meses que eu venho estudando Front-end e acredito que está na hora de ganhar mas um pouco de experiência. E @douglascorrea vou procurar investir mais tempo em projetos open source.

**Comment by douglascorrea** [comment]
@thiamsantos se quiser uma dica por onde começar, https://github.com/douglascorrea/react-hot-redux-firebase-starter, eu to precisando de uma ajudinha lá. Já separei as issues e acredito que seja fácil mandar uns PR pra fecha-las.

**Comment by willianjusten** [comment]
Como a maioria já comentou, eu reforço que como você está ainda sem cargas de responsabilidades ainda, aproveite ao máximo para estudar e se dedicar.

Utilize bastante o Github, escreva um blog talvez, faça experimentos para ir criando um portfólio. Assim vai ser muito mais proveitoso. Além disso, recomendo começar a se movimentar, ir em eventos, conversar com as pessoas da comunidade, o network é muito importante para quem deseja freelas, pois muitas vezes funciona por recomendação =)
